addons/project/security/project_security.xml

```diff
@@ -275,9 +275,7 @@
             ('project_id.privacy_visibility', '!=', 'followers'),
             '|',
                 ('project_id.message_partner_ids', 'in', [user.partner_id.id]),
-                '|',
-                    ('task_id.message_partner_ids', 'in', [user.partner_id.id]),
-                    ('user_ids', 'in', user.id),
+                ('user_ids', 'in', user.id),
```

Comment:
LGTM, however, we will have to update the ir.rule in a migration script for the 16.0 version. 🙂

Can you do that? 🙂 

PS: for the migration script, your branch name in `upgrade` repo should begin by `master-`. 🙂

Thanks in advance! 🙂 

@IT-Ideas FYI 🙂